docs/docs/segments/time.md

```diff
@@ -21,6 +21,19 @@ Show the current timestamp.
 }
 ```
 
+### Display time in multiple time zones with [Sprig](https://masterminds.github.io/sprig/date.html)
+
+```json
+{
+  "type": "time",
+  "properties": {
+    "time_format": "1/2 15:04"
+  },
+  "style": "plain",
+  "template": "\ue389 {{ .CurrentDate | date .Format }} {{ dateInZone \"15:04Z\" .CurrentDate \"UTC\" }}"
```

Comment:
I would only show the `template`, not the entire block.

Reply:
Would you show

```
"template": "\ue389 {{ .CurrentDate | date .Format }} {{ dateInZone \"15:04Z\" .CurrentDate \"UTC\" }}"
```
or skip icon and focus on template value (leave to user to Json-encode it correctly)? I think I prefer the latter slightly better, seems cleaner.
```
{{ .CurrentDate | date .Format }} {{ dateInZone "15:04Z" .CurrentDate "UTC" }}
```